src/ui/PreviewModal.tsx

```diff
@@ -16,20 +15,23 @@ interface Props {
   isOpen: boolean;
   onClose: (boolean) => void;
   isLandscape: boolean;
+  instance?: string;
```

Comment:
Could agency be more suitable name? And perhaps could this be refactored to boolean?

Reply:
Maybe. Do you mean that boolean might be e.g. isHSL?